docs/code_conventions/README.MD

```diff
@@ -194,25 +412,18 @@ def get_ip(ip):
 ## No Printing Sensitive Data To The Log
 This should go without saying, however when an integration is ready to be used as part of a public release (meaning you are done debugging it) we **always** remove print statements that are not absolutely necessary.
 
-## Catch Me If You Can
-As mentioned above, we only write code with 100% success rates, but we do come across surprise features quite often. Since we don't want a surprise to cause an issue, it is important to wrap your command block in a "Try-Catch" as demonstrated below:
-
-```python
-''' EXECUTION '''
-LOG('command is %s' % (demisto.command(), ))
-
-try:
-    if demisto.command() == 'test-module':
-        test_get_session()
-        demisto.results('ok')
 
-    if demisto.command() == 'atd-login':
-        get_session_command()
+## Dates
+We do not use epoch time for customer facing results (Context, Human Readable, etc.). If the API you are working with requires the time format to be in epoch, then convert the date string into epoch as needed. Where possible, use the human readable format of date ```%Y-%m-%dT%H:%M:%S```
 
-except Exception, e:
-    return_error(e.message)
+```python
+time_epoch = 499137720
+formatted_time = timestamp_to_datestring(time_epoch, "%Y-%m-%dT%H:%M:%S")
+print(formatted_time)
+>>> '1985-10-26T01:22:00'
 ```
 
+**Please note:** If the response returned is in epoch, it is best practices to convert it to ```%Y-%m-%dT%H:%M:%S```
```

Comment:
Not with %Z ?

Reply:
It was already written like this. But for human readable it is better without Z and milliseconds